### PR TITLE
omd.2.0.0.alpha

### PR DESCRIPTION
**Pull request by pw374**
let's see if it passes the tests...

### CODE REVIEW

File: packages/omd/omd.2.0.0.alpha [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+license: "ISC"
+homepage: "https://github.com/ocaml/omd"
+dev-repo: "git+https://github.com/ocaml/omd.git"
+bug-reports: "https://github.com/ocaml/omd/issues"
+tags: [ "org:ocamllabs" "org:mirage"  ]
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+build-test: [
```
Comment (mseri) [inline packages/omd/omd.2.0.0.alpha]: This has been renamed to `run-test` in opam2: https://opam.ocaml.org/doc/Manual.html#opamfield-run-test

Comment (pw374) [reply packages/omd/omd.2.0.0.alpha]: OK

**Comment by mseri** [comment]
I think you need to do `omd.2.0.0~alpha` if you want to be able to upgrade to 2.0.0 afterwards. Am I correct @kit-ty-kate ?

**Comment by kit-ty-kate** [comment]
@mseri yes that's exact

**Comment by kit-ty-kate** [comment]
Thanks!

**Comment by pw374** [comment]
> I think you need to do `omd.2.0.0~alpha` if you want to be able to upgrade to 2.0.0 afterwards. Am I correct @kit-ty-kate ?

I guess I can use omd.2.0.1 or something like that for the next release if omd.2.0.0.alpha can cause issues :)

**Comment by kit-ty-kate** [comment]
Don't worry, it's not an issue anymore (see my last commit on this PR). `omd.2.0.0` or even `omd.2.0.0.beta` will be fine

**Comment by pw374** [comment]
Oh I didn't realize you made the fix! Thanks :)

**Pull request merged**